pylot/debug/track_visualizer_operator.py

```diff
@@ -117,18 +117,40 @@ def _draw_trajectory_on_img(self, obstacle, segmented_frame, predict):
         intrinsic_matrix = segmented_frame.camera_setup.get_intrinsic_matrix()
 
         # Convert to screen points.
-        screen_points = [
-            transform.location.to_camera_view(extrinsic_matrix,
-                                              intrinsic_matrix)
-            for transform in obstacle.trajectory
-        ]
+        start_location = obstacle.bounding_box.transform.location - \
```

Comment:
Currently we do not support 3D obstacle detectors. Thus, obstacle.bounding_box is of type BoundingBox2D when we don't get the obstacles directly from Carla. It would be great to still be able to use the planner and the track visualizer even when we don't have access to 3D bonding boxes. Can you please update the code?

Reply:
@ICGog are the 2D bounding boxes top down? I thought those were camera 2d bounding boxes. In that case we wouldn't be able to directly visualize them in the `track_visualize_operator.py`

Reply:
Yes, we can't visualize them here. However, we should still be able to use this operator even if we can't visualize bounding boxes (i.e., it's useful to visualize the output from prediction). IMHO, I would just check if obstacle.bounding_box is of type BoundingBox3D, if it's not then I would not add any bounding boxes to the images.

Reply:
Got it, thanks for the clarification.